.github/workflows/main.yml

```
@@ -0,0 +1,92 @@
+name: CI
+
+on: pull_request
+
+jobs:
+  jjb:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python:
+          - "2.7"
+    name: Jenkins Job Builder
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup python
+        uses: actions/setup-python@v1
+        with:
+          python-version: ${{ matrix.python }}
+      - name: Restore pip cache
+        uses: actions/cache@v2
+        with:
+          path: ~/.cache/pip
+          key: ${{ runner.os }}-pip-${{ hashFiles('**/requirements*.txt') }}
+          restore-keys: |
+            ${{ runner.os }}-pip-
+      - name: Install dependencies
+        run: |
+          pip install jenkins-job-builder
+          pip install lxml
+          pip install requests
+      - name: Run tests
+        run: |
+          for d in puppet/modules/jenkins_job_builder/files/theforeman.org; do
+            DIR="."
+            ( cd $d && jenkins-jobs -l debug test -r $DIR )
+            ( cd $d && jenkins-jobs test -r $DIR --config-xml -o output && find output -name config.xml -exec python ../jenkins-lint.py --xml {} + && rm -rf output )
+          done
+
+          for d in puppet/modules/jenkins_job_builder/files/centos.org; do
+            DIR="jobs"
+            ( cd $d && jenkins-jobs -l debug test -r $DIR )
+            ( cd $d && jenkins-jobs test -r $DIR --config-xml -o output && find output -name config.xml -exec python ../jenkins-lint.py --xml {} + && rm -rf output )
+          done
+  puppet:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby:
+          - "2.4"
```

Puppet 6 includes Ruby 2.5.
```suggestion
          - "2.5"
```

It's also confusing to read `Puppet (2.4)` in the checks.